docs/user/plagiarism-check.rst

```diff
@@ -0,0 +1,174 @@
+
+  Ignore comparisons whose similarity is below this threshold.
+  A similarity of 0% means that there is no overlap between two submissions, a similarity of 50% means that about
+  half of two submissions are identical.
+  A similarity of 100% means that two submissions are completely identical.
+
+2. Minimum Score in % (minimum value 0, maximum value 100).
+
+  Consider only submissions with a score greater than or equal to this value.
+
+3. Minimum Size.
+
+  a. Default value: Consider only submissions whose size is greater or equal to this value.
+  b. Modeling exercises: Consider only submissions that have at least as many modeling elements as the specified value.
+  c. Text exercises: Consider only submissions that have at least as many words as the specified value.
+
+Actions
+^^^^^^^
+The user can use different actions to run plagiarism check, inspect the results or improve the performance.
+
+|plagiarism-actions1|
+
+.. note::
+        Plagiarism detection can take a long time for large courses.
+1. Detect plagiarism. The user can execute the plagiarism check using this action.
+
+2. Rerun plagiarism detection. The user can rerun the plagiarism check by executing this action. It can be helpful to check for plagiarism using the different settings as described in section :ref:`settings`.
+
+3. Clean up. The user can clean up the latest plagiarism check results for the selected exercise. It helps to keep the database slim and safe storage capacity. Moreover, it should improve the overall system performance. To execute the clean-up action, the user must approve it by clicking ok on the dialog, as this action deletes all potentially approved or denied plagiarism cases.
+
+|clean-up-dialog|
+
+4. Download. The user can download the plagiarism results in JSON or CSV format to open them in a selected editor for the further analysis.
+
+Results
+^^^^^^^
+After the plagiarism check was executed, the results can be inspected in different views.
+
+1. Overview of the similarity distribution. This statistical overview shows the similarity distribution based on the percentage as a histogram chart. The user can analyze the distribution quickly and adjust the plagiarism check settings as needed accordingly.
+ |run-results|
+2. Selected submission. When the user selects the submission from the submission list on the left side of the plagiarism run result view, new actions can be selected to initiate the plagiarism check workflow. We will provide further details to the workflow in the next section.
+ |run-results-selected-submissions|
+
+Plagiarism Check Workflow
+-------------------------
+
+In this section, we explain the process of the plagiarism case detection process from different users' perspective.
+
+Tutors
+^^^^^^
+The tutors are responsible for checking the plagiarism attempts of students. They execute the plagiarism checks and mark the potential plagiarism candidates to notify the instructors.
```

Comment:
```suggestion
Tutors and instructors can execute plagiarism checks. They carefully review the automatically identified cases and provide a first decision whether to accept or deny the found case.
```

Reply:
Rationale: otherwise is sounds that tutors have to do this, but there might be courses in which instructors are responsible for the whole process

Reply:
done